src/20_Components/amp-analytics.html

```diff
@@ -47,26 +59,47 @@
       justify-content: center;
       font-size: 20px;
       background: green;
-      color: white;
+      color: var(--color-text-light);
+    }
+    #testForm {
+      padding: var(--space-2);
+    }
+    #testForm div {
+      margin-bottom: var(--space-3);
+      position: relative;
+      display: inline-block
+    }
+    button, #testForm input[type=submit] {
+      text-transform: uppercase;
+      letter-spacing: .2em;
+      font-family: inherit;
+      font-weight: inherit;
+      font-size: var(--space-2);
+      line-height: 1.125rem;
+      margin: var(--space-2);
+      padding: var(--space-1);
+      text-decoration: none;
+      white-space: nowrap;
+      word-wrap: normal;
+      vertical-align: middle;
+      cursor: pointer;
+      border: 1px solid var(--color-primary);
+      background-color: var(--color-primary);
+      color: var(--color-text-light);
+    }
+    #testForm div {
+      padding: var(--space-0);
+      margin: var(--space-0);
+      max-width: 100%;
+      width: 300px;
+      min-width: 100px;
+      font-size: var(--space-1);
+      line-height:var(--space-2);
     }
     </style>
 </head>
 <body>
 
-  <!--
-  We are going to use a demo analytics endpoint to demonstrate what's possible with `amp-analytics`. The dashboard is embedded via `amp-iframe` and shows the data received by the analytics endpoint in real-time. It should be visible in the bottom right corner.
-  -->
-  <amp-iframe class="fixed-dashboard ampstart-card m1" width="180"
```

Comment:
Did you remove the iframe? That needs to stay.

Reply:
I added it back. With an empty comment it doesn' t show though.

Reply:
The dashboard doesn't show?